lib/sorcery.rb

```diff
@@ -1,5 +1,13 @@
 require 'sorcery/version'
 
+if RUBY_VERSION < '2.0.0'
+  warn '[DEPRECATION] Sorcery 0.10.0 will deprecate Ruby 1.9.3 and lower, please upgrade to Ruby 2.0 or higher.'
+end
+
+if Rails::VERSION < '4.0.0'
```

Comment:
Thanks @5t111111! It looks like that solved most of the issue, now it's just the test suite giving actual errors.